app/services/hyrax/admin_set_create_service.rb

```diff
@@ -195,7 +195,8 @@ def create!
       updated_admin_set = Hyrax.persister.save(resource: admin_set).tap do |result|
         if result
           ActiveRecord::Base.transaction do
-            permission_template = permissions_create_service.create_default(collection: result,
+            permission_template = PermissionTemplate.find_by(source_id: result.id.to_s) ||
+                                  permissions_create_service.create_default(collection: result,
```

Comment:
The transactions now create the permission_template.  But for ActiveFedora, it still needs to be created here.